### ISSUE TITLE
Rendere link non visibile

### ISSUE THREAD

**Issue by massi94** [Hexamini/PassioneKaraoke#83]
Secondo la WCAG [2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) #43 bisogna utilizzare un modo per fare il bypass di contenuti che si ripetono in ogni pagina per facilitare l'accesso a quelli che usano screen reader, bisogna quindi renderlo invisibile via CSS.
Nelle slide la gaggi dice di fare una cosa del genere:

``` css
.aiuti{
position:absolute;
height:0;
overflow:hidden
}
```

**Comment by Polpetta** [comment]
@massi94 La gaggi aveva detto di usare i link all'interno della pagina. Dici quelli?

**Comment by massi94** [comment]
@Polpetta Sisi scusate ho scritto male, il link interno lo ho già messo, con  la classe "aiuti" bisogna solo nasconderlo

**Comment by Polpetta** [comment]
@massi94 ok!

**Comment by WeissL** [comment]
@korut94 per disabilitare un link ci vuole una proprietà CSS che non ha nessun supporto per internet explorer < 11.... Facciamo in javascript?

**Comment by Polpetta** [comment]
@WeissL Lei disattiva Javascript.

**Comment by WeissL** [comment]
Non ci sono altri modi per farlo allora... a meno che non mettiamo un placeholder per Andrea per disabilitarli da Perl

**Comment by Polpetta** [comment]
@WeissL Bisogna sentire cosa dice @korut94

**Comment by korut94** [comment]
L'unica che mi viene in mente è creare una vista con il link. Poi si mette un segnaposto nel file page.html in corrispondenza dei link e io ci carico o il tag 'a' oppure solo il testo.

**Comment by WeissL** [comment]
@korut94 potresti inserire l'attributo disabled semplicemente :+1:

**Comment by korut94** [comment]
Ma se uno ha un browser che non supporta HTML5? Per gli iframe è un discorso di servizio, altrimenti il sito è completamente senza senzo. Invece la disabilitazione dei link è una funzionalità base per la Gaggi.

**Comment by WeissL** [comment]
disabled non è in css ed è supportato fino ad ie 6.. appena guardato

**Comment by korut94** [comment]
Sicuro sia per i link? L'ho trovato solo per i campi input.

**Comment by WeissL** [comment]
Vero non funzia >.< allora non so come fare >.<

**Comment by korut94** [comment]
Qualcuno domani fara' la view dei link.

**Comment by massi94** [comment]
@WeissL perche' vuoi disabilitare il link? non basta solo nasconderlo graficamente?

**Comment by WeissL** [comment]
perchè è brutto vedere che il menu cambia... la cosa più semplice era disabilitare il link...

**Comment by massi94** [comment]
non deve cambiare! il link li non va nel menu

**Comment by WeissL** [comment]
penso che Andrea intendesse le voci del menu da disabilitare ._.

**Comment by massi94** [comment]
ok, allora abbiamo fatto confusione, questa issue era aperta per una cosa diversa!

**Comment by WeissL** [comment]
la classe .aiuti è stata inserita nel css.. servono altre modifiche o chiudo?

**Comment by Polpetta** [comment]
@massi94 ti invoco
